tests/scripts/kubeadm-install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash +e
 
 KUBE_VERSION=${1:-"v1.13.1"}
-
+ARCH=$(dpkg --print-architecture)
```

Comment:
`dpkg`? This doesn't seem distro friendly...

Reply:
Currently, kubeadm-install only provides installation of Debian & Ubuntu, other versions such as centos are not supported.